### ISSUE TITLE
Doesn’t redirect on LibreWolf 89.0.0

### ISSUE THREAD

**Issue by gtdm** [kodxana/Watch-on-Odysee#2]
**Describe the bug**
Doesn't redirect to Odysee.

**To Reproduce**
1. Go to any YouTube video. 
2. See error. 

**Expected behavior**
Getting redirected.

**Desktop (please complete the following information):**
 - OS: [Gentoo Linux and Windows 10]
 - Browser [LibreWolf]
 - Version [89.0.0]

**Additional context**
I use several extensions (like uBlock Nightmare Mode) but Watch-on-Odysee works perfectly fine with the exact same settings in Brave.

**Comment by kodxana** [comment]
I tested on 89.0.2 (64-bit) Windows 11 and did not had any issues.
Can you post logs from console (CTRL + J )

**Comment by gtdm** [comment]
I've tested LibreWolf 89.0.2 on Windows 10 and it worked fine for me too. Looks like the issue is caused by the 89.0.0 release (I've changed the version number in the title and my initial post to avoid confusion).

This issue can be closed then, I guess, but just in case you still want to see the logs. These are the only things that pop up. Not sure if they're related though.

```
sendRemoveListener on closed conduit {884679b9-5d6b-48b2-90a7-15ae26ce568a}.274877907134 ConduitsChild.jsm:108
    _send resource://gre/modules/ConduitsChild.jsm:108
    _send self-hosted:1282
    removeListener resource://gre/modules/ExtensionChild.jsm:663
    removeListener resource://gre/modules/ExtensionChild.jsm:886
    onChanged chrome://extensions/content/child/ext-storage.js:332
    removeListener resource://gre/modules/ExtensionCommon.jsm:2528
    revoke resource://gre/modules/ExtensionCommon.jsm:2550
    close resource://gre/modules/ExtensionCommon.jsm:2555
    unload resource://gre/modules/ExtensionCommon.jsm:917
    unload resource://gre/modules/ExtensionPageChild.jsm:261
    unload resource://gre/modules/ExtensionPageChild.jsm:298
    destroyExtensionContext resource://gre/modules/ExtensionPageChild.jsm:482
    observe resource://gre/modules/ExtensionPageChild.jsm:395
```
```
[Exception... "Component returned failure code: 0x80520001 (NS_ERROR_FILE_UNRECOGNIZED_PATH) [nsIXPCComponents_Utils.readUTF8URI]"  nsresult: "0x80520001 (NS_ERROR_FILE_UNRECOGNIZED_PATH)"  location: "JS frame :: resource://gre/modules/L10nRegistry.jsm :: L10nRegistry.loadSync :: line 692"  data: no] L10nRegistry.jsm:692:19
    loadSync resource://gre/modules/L10nRegistry.jsm:692
    fetchFile resource://gre/modules/L10nRegistry.jsm:607
    generateResourceSetSync resource://gre/modules/L10nRegistry.jsm:512
    map self-hosted:224
    generateResourceSetSync resource://gre/modules/L10nRegistry.jsm:507
    generateResourceSetsForLocaleSync resource://gre/modules/L10nRegistry.jsm:449
    InterpretGeneratorResume self-hosted:1485
    next self-hosted:1433
    generateResourceSetsForLocaleSync resource://gre/modules/L10nRegistry.jsm:458
    InterpretGeneratorResume self-hosted:1485
    next self-hosted:1433
    generateBundlesSync resource://gre/modules/L10nRegistry.jsm:186
    InterpretGeneratorResume self-hosted:1485
    next self-hosted:1433
    next resource://gre/modules/Localization.jsm:147
    formatWithFallbackSync resource://gre/modules/Localization.jsm:287
    formatValuesSync resource://gre/modules/Localization.jsm:396
    formatValueSync resource://gre/modules/Localization.jsm:443
```

I've also noticed that the extension's icon in the menu bar doesn't show whether it's turned on or off. But this is also fixed on 89.0.2.
